database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,9 +15,17 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
-            $table->string('name');
+            $table->string('firstname');
+            $table->string('lastname');
             $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
```

Comment:
Is this for like activating the account?

Reply:
You can place also like is status or is active which can be a boolean we send a verification link to the user and on click it activates his/or her account

Reply:
I have added an "isactive" column on the users table